website/docs: revamp enterprise section (#20379)
goauthentik/authentik: website/docs/enterprise/enterprise-features.md

---
title: Enterprise features

> description: "The extra features that are included in authentik Enterprise"

Reviewer: ```suggestion
description: "Features which are included in authentik Enterprise"
```

---

This page describes the Enterprise features that are available in the authentik [Enterprise plans](https://goauthentik.io/pricing/).

## Enterprise vs Enterprise Plus

The Enterprise release of authentik is available in two tiers: Enterprise and Enterprise+.

Enterprise builds on authentik's open-source foundation with enterprise-grade features and ticket-based support.

> Enterprise+ is designed to provide the best overall end-user experience while mitigating risk and delivering the fastest time-to-value. It includes everything in Enterprise, plus onboarding best practices, dedicated support channels (Slack and/or scheduled video calls for complex issues) with SLA-backed response times, volume discounts, and support for FIPS-compliant deployments to meet FedRAMP-aligned requirements. It also supports billing and purchase via invoice.

Reviewer: ```suggestion
Enterprise Plus is designed to provide the best overall end-user experience while mitigating risk and delivering the fastest time-to-value. It includes everything in Enterprise, plus onboarding best practices, dedicated support channels (Slack and/or scheduled video calls for complex issues) with SLA-backed response times, volume discounts, and support for FIPS-compliant deployments to meet FedRAMP-aligned requirements. It also supports billing and purchase via invoice.
```

to match the rest and the pricing page

---
title: Enterprise features
description: "The extra features that are included in authentik Enterprise"
---

> This page describes the enterprise features that are available in the authentik [Enterprise plans](https://goauthentik.io/pricing/).

Reviewer 2: Let's stick with uppercase for Enterprise... we are that cool, and the word is uppercase lower down on this same page.
